src/main/java/application/controller/BookController.java

```diff
@@ -26,8 +26,10 @@
 @Tag(name = "Book management", description = "Endpoints for book management")
 @RequiredArgsConstructor
 @RestController
-@RequestMapping("/api/books")
+@RequestMapping(BookController.BASE_URL)
 public class BookController {
+    public static final String BASE_URL = "/api/books";
+    public static final String SEARCH_URL = "/search";

```

Comment:
It is weird to add constants if the task is to add tests.
I saw Kostia`s comment, but pay attention that it is regarding tests.